components/MonitoringUserSetupInstructions.tsx

```diff
@@ -172,52 +152,8 @@ $$
   /* pganalyze-collector */ SELECT * FROM pg_catalog.pg_stat_replication;
 $$ LANGUAGE sql VOLATILE SECURITY DEFINER;`}
       </CodeBlock>
-    </>
-  )
-}
-
-const MonitoringUser96: React.FunctionComponent<{password: string}> = ({password}) => {
-  const CodeBlock = useCodeBlock();
-  return (
-    <>
-      <CodeBlock>
-        {`CREATE USER pganalyze WITH PASSWORD '${password}' CONNECTION LIMIT 5;
-
-CREATE SCHEMA pganalyze;
-GRANT USAGE ON SCHEMA pganalyze TO pganalyze;
-GRANT USAGE ON SCHEMA public TO pganalyze;
-
-CREATE EXTENSION IF NOT EXISTS pg_stat_statements;
-
-CREATE OR REPLACE FUNCTION pganalyze.get_stat_statements(showtext boolean = true) RETURNS SETOF pg_stat_statements AS
-$$
-  /* pganalyze-collector */ SELECT * FROM public.pg_stat_statements(showtext);
-$$ LANGUAGE sql VOLATILE SECURITY DEFINER;
-
-CREATE OR REPLACE FUNCTION pganalyze.get_stat_activity() RETURNS SETOF pg_stat_activity AS
-$$
-  /* pganalyze-collector */ SELECT * FROM pg_catalog.pg_stat_activity;
-$$ LANGUAGE sql VOLATILE SECURITY DEFINER;
-
-CREATE OR REPLACE FUNCTION pganalyze.get_column_stats() RETURNS SETOF pg_stats AS
-$$
-  /* pganalyze-collector */ SELECT schemaname, tablename, attname, inherited, null_frac, avg_width,
-  n_distinct, NULL::anyarray, most_common_freqs, NULL::anyarray, correlation, NULL::anyarray,
-  most_common_elem_freqs, elem_count_histogram
-  FROM pg_catalog.pg_stats;
-$$ LANGUAGE sql VOLATILE SECURITY DEFINER;
-
-CREATE OR REPLACE FUNCTION pganalyze.get_stat_replication() RETURNS SETOF pg_stat_replication AS
-$$
-  /* pganalyze-collector */ SELECT * FROM pg_catalog.pg_stat_replication;
-$$ LANGUAGE sql VOLATILE SECURITY DEFINER;
-
-CREATE OR REPLACE FUNCTION pganalyze.get_stat_progress_vacuum() RETURNS SETOF pg_stat_progress_vacuum AS
-$$
-  /* pganalyze-collector */ SELECT * FROM pg_catalog.pg_stat_progress_vacuum;
-$$ LANGUAGE sql VOLATILE SECURITY DEFINER;`}</CodeBlock>
```

Comment:
FWIW, I think it may still be useful to describe these somewhere in our docs (since not all platforms allow pg_monitor access), but it doesn't have to be in this component (which is used in the setup guide for supported platforms), so makes sense to remove for now.

Reply:
Makes sense. If nothing else, we could have a troubleshooting section on monitoring without `pg_monitor`.